### DISCUSSION TITLE
Meshtastic on a T-embed

### DISCUSSION THREAD

**Discussion by AlanB74** [meshtastic/Discussions#95] [Firmware]
Hey, I'm ordering a Lilygo T-embed CC1101 for another project but noticed whispers on Reddit (I think) about some kind of LoRa utilisation 🤔 anybody know about this? Or is there a version of Meshtastic that may be compatible with it? Interested to find out

**Comment by FOXIK-Lab** [comment]
Hi, I'm telling the truth in the documentation, there are hints about the lore, but in fact it's missing. If you want, you can solder it. I'll try to make a port now, but I don't think I'll succeed.
